packs/data/other-effects.db/effect-2-circumstance-penalty-to-ranged-attacks.json

```
@@ -0,0 +1,50 @@
+{
+    "_id": "wVGSlZEGxsg1s8AZ",
+    "img": "systems/pf2e/icons/effects/crtical-hit-fumble.webp",
+    "name": "Effect: -2 circumstance penalty to ranged attacks",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by Critical Hit/Fumble Deck<br /></strong>-2 circumstance penalty to ranged attacks</p>"
```

```suggestion
            "value": "<p>Effect applied by Critical Hit/Fumble Deck. -2 circumstance penalty to ranged attacks</p>"
```
We don't use `<br>` tags, we'd just use new paragraphs instead.

Maybe we should say what cards it's granted by? Something like:

`Granted by Critical Hit Deck cards 17, 34.`

---

Fair enough, I think this was the copy pasta into the editor. I can change all these.
And maybe not a bad idea to reference back to all the cards it's used in.

I wasn't sure if we even wanted/needed description text but after adding to the compendium it felt that it needed some explanation. I'll give these a brush up.